unitTestSupportFunctions/getBodyNodes.m

```diff
@@ -0,0 +1,62 @@
+%% Function documentation
+%
+% Returns all non-boundary nodes that have boundary conditions set on them
+% Works only for 2D rectangular meshes 
+%
+%               Input :
+%              fldMsh : Nodes and elements for the fluid mesh
+%              homDBC : The global numbering of the DOFs where homogeneous
+%                       Dirichlet boundary conditions are applied
+%
+%              Output :
+%    nodalCoordinates : Nodes and elements from the fluid mesh that lie on
+%                       a body 
+%             homDOFs : The global numbering of the DOFs that lie on a body
+%                       where homogeneous Dirichlet boundary conditions are
+%                       applied
+%
+%% Function main body
+function homDOFs = getBodyNodes (fldMsh, homDBC)
```

Comment:
This function seems to return the numbering of all DOFs which are on the bounday of a rectangular domain. This means that this function is not general and thus the employed name misleading. Use instead of getBodyNodes the name getBodyNodesOnRectangularDomain2D